src/analysis/sum_integral_comparisons.lean

```diff
@@ -0,0 +1,169 @@
+/-
+Copyright (c) 2022 Kevin H. Wilson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kevin H. Wilson
+-/
+import measure_theory.measure.measure_space_def
```

Comment:
This is transitively imported by the next import
```suggestion
```

Reply:
Is there a way to determine this automatically other than trial and error? Would be happy to minimize future imports!

Reply:
I think someone has a linter that does that somewhere (I don't remember who), but we don't apply it systematically, and I read that in some files there are "useless" imports that are intentionally left in to insist on the fact that the imported notion is needed.

In the present case `measure_space_def` is such a basic file in measure theory that I just knew that it had to be imported by the next line.